tests/test_enhancements.py

```diff
@@ -0,0 +1,122 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+def test_get_by_newest():
```

Comment:
Nice job adding tests for your age-related functions. One thing I'd be a little careful of is that for most of these tests, the newest item happens to be the first thing in the inventory. Try having the newest at a more varied distribution of locations. Until the very final test, I think most of these tests would pass if `get_newest_item` simply returned the first thing in the inventory.